src/coreclr/src/vm/processdiagnosticsprotocolhelper.h

```diff
@@ -18,8 +18,9 @@ class IpcStream;
 // see diagnosticsipc.h and diagnosticserver.h for more details
 enum class ProcessCommandId : uint8_t
 {
-    GetProcessInfo = 0x00,
-    ResumeRuntime  = 0x01,
+    GetProcessInfo        = 0x00,
+    ResumeRuntime         = 0x01,
+    GetProcessEnvironment = 0x02
```

Comment:
Is there a companion PR to the diagnostics repo with the spec update?

Reply:
There will be.  It hasn't been created yet.